addons/mail/static/tests/qunit_suite_tests/components/activity_tests.js

```diff
@@ -734,8 +749,9 @@ QUnit.test('activity upload document is available', async function (assert) {
         1,
         "should have activity upload button"
     );
-    assert.ok(
-        chatterContainerComponent.chatter.activityBoxView.activityViews[0].fileUploader,
+    assert.containsOnce(
+        document.body,
+        '.o_ChatterTopbar_buttonAttachments',
```

Comment:
This is not a correct change, the activity upload is a specific thing that was covered by this test.